go.mod

```diff
@@ -78,7 +78,8 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/btcsuite/btcd/btcec/v2 v2.3.3 // indirect
 	github.com/cespare/xxhash v1.1.0 // indirect
-	github.com/cespare/xxhash/v2 v2.3.0 // indirect
+	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/cockroachdb/apd/v2 v2.0.2 // indirect
```

Comment:
Curious why we landed on apd v2 instead of v3? Didn't see a discussion in either of https://github.com/cosmos/cosmos-sdk/issues/7773#issuecomment-1026106336 or https://github.com/cosmos/cosmos-sdk/issues/11783 , just a brief mention. In some testing we did it showed 20-30% better performance improvements in overall runtime.

Reply:
This just looks like a mistake